package.json

```diff
@@ -7,16 +7,18 @@
     "url": "git+https://github.com/danawoodman/react-fontawesome.git"
   },
   "config": {
-    "entry": "src/index.js"
+    "entry": "src",
+    "output": "lib",
+    "mainFile": "src/index.js"
   },
   "bugs": {
     "url": "https://github.com/danawoodman/react-fontawesome/issues"
   },
   "main": "lib/index.js",
   "scripts": {
-    "build": "babel -o $npm_package_main $npm_package_config_entry",
+    "build": "babel $npm_package_config_entry -o $npm_package_config_output",
```

Comment:
If you run babel through an npm run script it will use the one in `node_modules`.

> In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts.

Source: https://docs.npmjs.com/cli/run-script#description